.github/workflows/ci.yml

```diff
@@ -9,37 +9,42 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Fetch the src
-      uses: actions/checkout@v3
-    - name: Set up Python 3.11
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.11
+      - name: Fetch the src
+        uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
 
-    - name: Install requirements
-      run: python -m pip install -r requirements/dev.in -c requirements/dev.txt
+      - name: Install requirements
+        run: python -m pip install -r requirements/dev.in -c requirements/dev.txt
 
-    - name: Run tests
-      run: python -m manage test
+      - name: Run tests
+        run: python -m coverage run manage.py test -v 2
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This token is unnecessary for public repos.
```suggestion
```

Reply:
I guess you can disregard this per https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/21#issuecomment-1723553104.